e2e/tests/functional/plugins/plot/plotControls.e2e.spec.js

```diff
@@ -108,4 +108,59 @@ test.describe('Plot Controls', () => {
     // Expect before and after plot points to match
     await expect(plotPixelSizeAtPause).toEqual(plotPixelSizeAfterWait);
   });
+
+  test('Plots follow the right time context', async ({ page }) => {
+    // Set realtime mode with 2 second window
+    const startOffset = {
+      startMins: '00',
+      startSecs: '01'
+    };
+
+    const endOffset = {
+      endMins: '00',
+      endSecs: '01'
+    };
+
+    // Switch to real-time mode
+    await setRealTimeMode(page);
+
+    // Set start time offset
+    await setStartOffset(page, startOffset);
+
+    // Set end time offset
+    await setEndOffset(page, endOffset);
+
+    // hover over plot for plot controls
+    await page.getByLabel('Plot Canvas').hover();
+    // click on pause control
+    await page.getByTitle('Pause incoming real-time data').click();
+    // expect plot to be paused
+    await expect(page.getByTitle('Resume displaying real-time data')).toBeVisible();
+    // synchronize time conductor
+    await page.getByTitle('Synchronize Time Conductor').click();
+
+    // expect warning message, ack it.
+    await expect(page.getByLabel('Modal Overlay')).toBeVisible();
+    await page.getByLabel('Modal Overlay').getByRole('button', { name: 'OK' }).click();
+
+    // Toggle independent time conductor ON
+    await page.getByLabel('Enable Independent Time Conductor').click();
+    // Toggle independent time conductor OFF
```

Comment:
Before toggling independent time conductor off we need to ensure it is in fixed mode. If both the global time conductor and independent time conductor modes are realtime, you won't see this bug symptom, even without the bug fix.

Reply:
Good point. Done.